R/readQFeatures.R

```diff
@@ -65,20 +69,65 @@
 ##' |   .    | .    | ...            | ...       |
 ##' |--------+------+----------------+-----------|
 ##' ```
-##'
-##' Whe using a `colData` `data.frame` as input, it must contain at
-##' least `quantCols` and `runCol` (in the multi-set case)
-##' variables. Additional sample annotations are desirable, and the
-##' reason such a `colData` can be passed directly to populate the
-##' `colData` slot of the returned object. When using the `quantCols`
-##' and `runCol` only, the `colData` slot contains zero
+##' 
+##' **Why splitting the table into multiple sets?** In the table 
+##' depicted above, the quantCols provide quantitative values for
+##' multiple sample (one sample in each run/batch provided by 
+##' `runCol`). This violates the `SummarizedExperiment` paradigm where
+##' each column should contain an individual sample and would make 
+##' downstream data manipulation and data analyses cumbersome. A
+##' solution could be to reshape the table into a wide format to
+##' ensure that every sample is contained in a separate column. 
+##' However, this would assume that rows of the table are shared 
+##' across runs. When working with PSM data, every PSM is 
+##' uniquely characterised by the run it is found. Hence, there are no
+##' shared feature identifiers across runs that would enable a sound
+##' widening of the table. Splitting the table into sets is the only 
+##' solution for having every sample in a separate column while 
+##' preserving the PSM identity.
```

Comment:
I am not convinced by this explanation. It is correct, of course, but it makes reference to technical terms like `SummarziedExperiment`, that don't necessarily help with understanding the why. 

I am not sure I will have time to rephrase it today, but the emphasis should be on the experimental design and the data acquisition.